L1Trigger/TrackFindingTMTT/src/Settings.cc

```diff
@@ -42,6 +42,7 @@ namespace tmtt {
         //kalmanDebugLevel_(2), // Good for debugging
         kalmanMinNumStubs_(4),
         kalmanMaxNumStubs_(6),
+        kalmanAddBeamConstr_(false),
```

Comment:
Add comment, explaining this parameter. e.g. "// Results from 5-param fit have post-fit beam-spot constraint applied".

Reply:
I included a comment